components/skus/browser/rs/wasm/src/lib.rs

```diff
@@ -78,10 +78,7 @@ pub fn initialize(
             .with_writer(MakeConsoleWriter);
         let perf_layer = performance_layer().with_details_from_fields(Pretty::default());
 
-        tracing_subscriber::registry()
-            .with(fmt_layer)
-            .with(perf_layer)
-            .init();
+        tracing_subscriber::registry().with(fmt_layer).with(perf_layer).init();
```

Comment:
pls revert this unrelated change. You might need to do `git fetch origin master` locally.

Reply:
Nice catch - didn't notice when I formatted it!